docs/plans/2026-05-06-feat-archscope-agent-review-inevitability-plan.md

```
@@ -0,0 +1,944 @@
+source_spec: docs/specs/2026-05-06-feat-archscope-agent-review-inevitability-spec.md
+plan_route: fresh
+plan_depth: deep
+traceability_required: true
+tracking_status: tracked
+linear_status: created
+linear_issue: JSC-280
+linear_url: https://linear.app/jscraik/issue/JSC-280/make-archscope-inevitable-for-coding-agents-and-pr-reviewers
+linear_team: Jscraik
+linear_priority: Medium
+branch: jscraik/jsc-280-make-archscope-inevitable-for-coding-agents-and-pr-reviewers
+pr: pending
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)